api/routes/apds/submissions/openApi.js

```diff
@@ -0,0 +1,67 @@
+const {
+  schema: { arrayOf, jsonResponse }
+} = require('../../openAPI/helpers');
+
+const openAPI = {
+  '/apds/submissions': {
+    get: {
+      tags: ['APDs'],
+      summary: 'Get all of the APDs that have been submitted',
+      description: 'Get a list of all APDs that have been submitted',
+      // security: {
```

Comment:
What's the plan with the?

Reply:
eventually we will have an api key secure this endpoint but that relies on getting everything working with kong